apis/openapi/elevenlabs.io/main/1.0/openapi.json

```
@@ -4,7 +4,7 @@
     "title": "ElevenLabs API Documentation",
     "description": "This is the documentation for the ElevenLabs API. You can use this API to use our service programmatically, this is done by using your xi-api-key. <br/> You can view your xi-api-key using the 'Profile' tab on https://elevenlabs.io. Our API is experimental so all endpoints are subject to change.",
     "version": "1.0",
-    "x-jentic-source-url": "https://api.elevenlabs.io/openapi.json"
+    "x-jentic-source-url": "https://raw.githubusercontent.com/jentic/jentic-public-apis/refs/heads/main/apis/openapi/elevenlabs.io/main/1.0/openapi.json"
```

```suggestion
    "x-jentic-source-url": "https://api.elevenlabs.io/openapi.json"
```

Can apply this suggestion to keep the original source intact